src/renderers/webgpu/WebGPUBackend.js

```diff
@@ -695,45 +696,20 @@ class WebGPUBackend extends Backend {
 
 		} else {
 
-			supportsDepth = renderTargetData.depth;
-			supportsStencil = renderTargetData.stencil;
+			supportsDepth = renderTargetContext.depth;
+			supportsStencil = renderTargetContext.stencil;
 
 			if ( color ) {
 
-				for ( const texture of renderTargetData.textures ) {
+				const descriptor = this._getRenderPassDescriptor( renderTargetContext );
```

Comment:
@sunag FYI here I updated the RenderContext to support RenderTarget that has not been rendered to the scene yet.

This way this prevent issues where the clear Pass doesn't match the correct descriptor associated to a texture and reuse how we setup a descriptor with `_getRenderPassDescriptor`.